build/commands/lib/config.js

```diff
@@ -258,6 +258,22 @@ Config.prototype.buildArgs = function () {
     ...this.extraGnArgs,
   }
 
+  if (process.platform === 'darwin') {
+    // TODO(petemill): Identify "brave corporate" developers via an env variable
+    // or .npmrc flag and download the SDK for them, always setting
+    // use_system_xcode = false unless specified otherwise via a flag.
+    // For now we're only doing this for `use_goma` which means when developers
+    // do not specify this flag, their build output will be invalidated and
+    // will likely have to do a full re-build.
+    if (this.use_goma) {
+      // SDK Version is consistent for all users so we can share distributed
+      // cache.
+      const sdkVersion = "11.1"
+      args.use_system_xcode = false
+      args.mac_sdk_official_version = sdkVersion
```

Comment:
we don't need or want to set this at all. We want to use the default in upstream which is 11.1 and when upstream changes we want to change with it

Reply:
ok apologies I just realized that this variable is always set upstream. And when we provide a package we'll just have to version it according to our branch or something similar.

From upstream:
```
  # The SDK version used when making official builds. This is a single exact
  # version, not a minimum. If this version isn't available official builds
  # will fail.
  mac_sdk_official_version = "11.1"
```
and when `use_system_xcode == false`:
```
mac_sdk_version = mac_sdk_official_version
```